src/main/target/FLYWOOF411/target.h

```diff
@@ -127,7 +124,7 @@
 
 #define USE_LED_STRIP
 
-#define ENABLE_DSHOT_DMAR       DSHOT_DMAR_ON									
+#define ENABLE_DSHOT_DMAR       DSHOT_DMAR_ON                                   
```

Comment:
👾

Reply:
When I commit I run a script to convert tabs to spaces. Ideally there would be no trailing spaces either, but they're better than tabs.

Reply:
Fixed